.dockerignore

```diff
@@ -1,5 +1,4 @@
 bin/
 .idea
 .github
-testdata/
```

Comment:
docker build doesn't perform tests, therefore this directory is excluded to minimize the amount of transferred data to the docker daemon. Did you need this directory for your build?

Reply:
Needed it for local testing (Dockerfile.Test)